packages/core/src/components/table/table/table.scss

```diff
@@ -11,3 +11,19 @@
 .tds-table--responsive {
   width: 100%;
 }
+
+.tds-table--horizontall-scroll {
+  display: block;
+  overflow-x: scroll;
+  white-space: nowrap;
+}
+
+::-webkit-scrollbar {
```

Comment:
Just a thought, we have a `_scrollbar.scss`
https://github.com/scania-digital-design-system/tegel/blob/a1b37a6f411b5d896e9883a2460f86c90a59e187/packages/core/src/mixins/_scrollbar.scss#L2

that just supports vertical scroll. Maybe we could build upon it and create a horizontal one also?
Something like:
`@mixin tds-scrollbar($orientation: vertical) { }`